soc/silabs/silabs_siwx91x/Kconfig

```diff
@@ -14,3 +14,19 @@ config SOC_SILABS_SLEEPTIMER
 	bool
 	help
 	  The Sleeptimer HAL module is used for SIWX91X.
+
+config SIWX91X_SILABS_COMMANDER_SIGN
+	bool "Silicon Labs Commander Signing Support"
+	depends on BUILD_OUTPUT_BIN
+	help
+		Activates the signing process for the Zephyr binary using Silicon Labs Commander during the build process.
+
+config M4_OTA_KEY
```

Comment:
These options should be `SIWX91X_OTA_KEY` and `SIWX91X_PRIVATE_KEY` in my opinion, in order to be properly namespaced.